### ISSUE TITLE
About MillionAID dataset

### ISSUE THREAD

**Issue by pUmpKin-Co** [ViTAE-Transformer/ViTAE-Transformer-Remote-Sensing#3]
I downloaded the MillionAID dataset from the official homepage: [MillionAID](https://captain-whu.github.io/DiRS/). The training set was found to have only 10K images. The test set is not labeled. May I know how the pre-training data in the paper was obtained?

**Comment by DotWang** [comment]
The fourth author in our paper is the leader of the MillionAID project.

**Comment by DotWang** [comment]
The original split of MillionAID is used for recognition. Our study is about pretraining,  so we resplit the training and testing sets. The obtained training set is relatively large for transferring the pretrained weights to downstream tasks. All RSP pretrained weights are available at https://github.com/ViTAE-Transformer/ViTAE-Transformer-Remote-Sensing/blob/main/README.md

**Comment by pUmpKin-Co** [comment]
> The original split of MillionAID is used for recognition. Our study is about pretraining, so we resplit the training and testing sets. The obtained training set is relatively large for transferring the pretrained weights to downstream tasks. All RSP pretrained weights are available at https://github.com/ViTAE-Transformer/ViTAE-Transformer-Remote-Sensing/blob/main/README.md

Thank you very much. Your work is very inspiring to us. We are working to do some research in the field of remote sensing image pre-training and would like to use your work as a baseline. However, we found that the open source MillionAID data has less annotated data than in your paper, so we propose this issue.

**Comment by redsnappper** [comment]
> > The original split of MillionAID is used for recognition. Our study is about pretraining, so we resplit the training and testing sets. The obtained training set is relatively large for transferring the pretrained weights to downstream tasks. All RSP pretrained weights are available at https://github.com/ViTAE-Transformer/ViTAE-Transformer-Remote-Sensing/blob/main/README.md
> 
> Thank you very much. Your work is very inspiring to us. We are working to do some research in the field of remote sensing image pre-training and would like to use your work as a baseline. However, we found that the open source MillionAID data has less annotated data than in your paper, so we propose this issue.

---

Hello,
I wasn't able to understand what the conclusion is. 
> > we resplit the training and testing sets.

I think the annotated data is needed to use all images and split the images again. Is the labeled data provided for test data?  I mean, in my understanding, we need both "train_label.txt" and "valid_label.txt" to use million-AID, but I don't know where I can download them. I appreciate your help.

**Comment by chaoer** [comment]
> The original split of MillionAID is used for recognition. Our study is about pretraining, so we resplit the training and testing sets. The obtained training set is relatively large for transferring the pretrained weights to downstream tasks. All RSP pretrained weights are available at https://github.com/ViTAE-Transformer/ViTAE-Transformer-Remote-Sensing/blob/main/README.md

As you said in the paper, "we randomly choose 1,000 images in each category of the MillionAID dataset to form the validation set"

Could you please provide more details about how to do this selection? 
e.g. Is there any private label info for the test dataset of MillionAID ?
